Capstone_Project/Data_StoryTelling_EDA.ipynb

```diff
@@ -0,0 +1,2588 @@
+{
```

Comment:
@nishalpattan To make this easier, please add a high-level description of what is the goal (goals) of this notebook. Giving some context makes things easier to the reviewer